src/pipx/main.py

```diff
@@ -57,7 +57,7 @@ def print_version() -> None:
     Virtual Environment location is {str(constants.PIPX_LOCAL_VENVS)}.
     Symlinks to apps are placed in {str(constants.LOCAL_BIN_DIR)}.
 
-    """
+    """.replace(str(Path.home()), "$HOME")
```

Comment:
Would this make sense on Windows too? I'm not a big fan of this change; I prefer the help message to show the actual paths, not some placeholder values.

Reply:
It will *work* on Windows, but `$HOME` isn't expected to be set on Windows.

Reply:
That's what I mean. This change only makes sense on Debian. Overall I'm -1 on this change proposal. It's much better to show the path in the help message than add some placeholder value.

Reply:
Alternatively we could do the replacement in #744, the main motivation here was to make the manpages reproduceable.

There is a similar mechanism in generate_docs, already: https://github.com/pypa/pipx/blob/main/scripts/generate_docs.py#L23